### DISCUSSION TITLE
Make tasmota search longer for the wlan-server

### DISCUSSION THREAD

**Discussion by hansjeu** [arendst/Tasmota#22659] [Q&A] [answered]
A friend stop his wlan at night and start it in the morning.
The trouble is that his tasmota devices lose contact and he has te restart them all by hand.
Is there a way to tell the tasmota devices to just wait until the wlan server is runnng ageain.
Or maybee tasmota devices havr te wait 12 hours before resetting themsefs?

Thans hansjeu

**Comment by Jason2866** [comment, marked as answer]
Already there and default setting `WifiConfig 4` or `WifiConfig 5` See https://tasmota.github.io/docs/Commands/#wi-fi

**Reply by hansjeu** [reply]
Thank you. I will try them both this week.
